app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PostsController < ApplicationController
+  def index; end
+
+  def show; end
+end
```

Comment:
- [ ] `README.md` file: As required for all projects to make your README professional, I think it would be better if you update your `Getting Started` section with correct information such as of how to `access` the cloned directory,  add the `correct` cloning link, `install` your app, `use/run` your app, `run tests`(if they exist),... Most of information you added are wrong, as they are not related to this project. This will make your documentation to be professional and more descriptive. **For example to help you, just refer this [article](https://gist.github.com/tresorsawasawa/4cde69cff1b40161b9de380f9891f415)**